bootstrap/examples/kubeflow_toolbox.yaml

```diff
@@ -0,0 +1,66 @@
+# Deploy kubeflow to namespace "kubeflow"
+
+# TODO(inc0): Create rbac role bindings
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubeflow
+
+# Headless service because StatefulSet requires it
```

Comment:
I thought we only needed a service for a stateful set if we wanted to associate stable DNS names for them.

Reply:
Yeah, but to deploy statefulset you need a service. So I had to create this mock service. Look at servicename field in statefulset definition, it won't run without it.